test/parallel/test-fs-mkstemp.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+
+const tmpdir = require('../common/tmpdir');
+tmpdir.refresh();
+
+const tmpFile = fs.mkstempSync(path.join(tmpdir.path, 'foo.'));
+
+assert.match(path.basename(tmpFile.path), /^foo\.[0-9a-z]{6}$/i);
```

Comment:
I see in the failure that `_` is another possible character.
Is there a reliable list or should I just assert that there are 6 chars?

Reply:
I think just checking for 6 characters should be adequate.